### DISCUSSION TITLE
Unknown flashrom when I try to write GB/GBC game

### DISCUSSION THREAD

**Discussion by rafaliyo** [sanni/cartreader#166] [Need help]
Hi I had the cartreader with the 5.5 version and I have update it to the 6.1 version but now, when I try to flash a GB or GBC rom in my cartridge I always get:

Flash ID: BBB9 (this always is different)
Unkown flashrom

Press Button...

Before that I have no problems to flash the memory (AM29F016B)

When I upgraded I forgot to replace gba.txt file but I have done and I still get the same error. Is there anything I have to change?

**Comment by sanni** [comment]
As far as I can remember nothing has changed concerning flashing GB cartridges between 5.5 and 6.1. 
Check that the voltage switch is set to 5V.
Can you go back to 5.5 to see if that fixes the issue?
The only noteworthy change was going from FAT32 to exFAT format for the SD card but that should not make a difference here.

**Reply by rafaliyo** [reply]
I dont know how but it works now. I have done nothing. Just charging the battery of the cart reader that was low......anyway thanks for your help

**Reply by sanni** [reply]
Low battery would explain the issue. Glad you got it working.
